runelite-client/src/main/java/net/runelite/client/plugins/friendlist/FriendListPlugin.java

```diff
@@ -62,44 +63,52 @@ protected void shutDown()
 	}
 
 	@Subscribe
-	public void onGameTick(GameTick tick)
+	public void onScriptPostFired(ScriptPostFired event)
 	{
-		final int world = client.getWorld();
-		final boolean isMember = client.getVar(VarPlayer.MEMBERSHIP_DAYS) > 0;
-
-		final NameableContainer<Friend> friendContainer = client.getFriendContainer();
-		final int friendCount = friendContainer.getCount();
-		if (friendCount >= 0)
+		if (event.getScriptId() == ScriptID.FRIENDS_LIST_BUILD)
 		{
-			final int limit = isMember ? MAX_FRIENDS_P2P : MAX_FRIENDS_F2P;
+			final int world = client.getWorld();
+			final boolean isMember = client.getVar(VarPlayer.MEMBERSHIP_DAYS) > 0;
+			final NameableContainer<Friend> friendContainer = client.getFriendContainer();
+			final int friendCount = friendContainer.getCount();
+			if (friendCount >= 0)
+			{
+				final int limit = isMember ? MAX_FRIENDS_P2P : MAX_FRIENDS_F2P;
 
-			final String title = "Friends - W" +
-				world +
-				" (" +
-				friendCount +
-				"/" +
-				limit +
-				")";
+				final String title = "Friends - W" +
+					world +
+					" (" +
+					friendCount +
+					"/" +
+					limit +
+					")";
 
-			setFriendsListTitle(title);
+				setFriendsListTitle(title);
+			}
 		}
-
-		final NameableContainer<Ignore> ignoreContainer = client.getIgnoreContainer();
-		final int ignoreCount = ignoreContainer.getCount();
-		if (ignoreCount >= 0)
+		else if (event.getScriptId() == ScriptID.IGNORE_LIST_BUILD)
 		{
-			final int limit = isMember ? MAX_IGNORES_P2P : MAX_IGNORES_F2P;
+			final int world = client.getWorld();
+			final boolean isMember = client.getVar(VarPlayer.MEMBERSHIP_DAYS) > 0;
+			final NameableContainer<Ignore> ignoreContainer = client.getIgnoreContainer();
+			final int ignoreCount = ignoreContainer.getCount();
+			if (ignoreCount >= 0)
+			{
+				final int limit = isMember ? MAX_IGNORES_P2P : MAX_IGNORES_F2P;
 
-			final String title = "Ignores - W" +
-				world +
-				" (" +
-				ignoreCount +
-				"/" +
-				limit +
-				")";
+				final String title = "Ignores - W" +
+					world +
+					" (" +
+					ignoreCount +
+					"/" +
+					limit +
+					")";
 
-			setIgnoreListTitle(title);
+				setIgnoreListTitle(title);
+			}
 		}
+
+
```

Comment:
There's already extra whitespace at the end of the method before, we definitely don't need more of it. Remove these empty lines.
```suggestion
```